src/Modules/AuthenticationModule/components/ChangePass/ChangePass.tsx

```diff
@@ -0,0 +1,173 @@
+import { Visibility, VisibilityOff } from "@mui/icons-material";
+import {
+  Alert,
+  Box,
+  Button,
+  FormControl,
+  IconButton,
+  InputAdornment,
+  InputLabel,
+  OutlinedInput,
+  Typography,
+} from "@mui/material";
+import { useEffect, useRef, useState } from "react";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { ChangePasswordProps } from "../../../../interfaces/interface";
+import { userRequest } from "../../../../utils/request";
+import { getErrorMessage } from "../../../../utils/error";
+import { useToast } from "../../../Context/ToastContext";
+import { useNavigate } from "react-router-dom";
+
+export default function ChangePass() {
+  const [oldPass, setOldPass] = useState(false);
+  const [newPass, setNewPass] = useState(false);
+  const [confirmPass, setConfirmPass] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const navigate = useNavigate()
+  const timeoutRef = useRef<number | null>(null);
+  const { showToast } = useToast();
+
+  const {
+    register,
+    handleSubmit,
+    getValues,
+    formState: { errors },
+  } = useForm<ChangePasswordProps>();
+
+  const onSubmit: SubmitHandler<ChangePasswordProps> = async (data) => {
+    setLoading(true);
+    try {
+      const res = await userRequest.post("/admin/users/change-password", data);
+      showToast("success", res.data.message);
+      timeoutRef.current = setTimeout(()=>{
+        navigate("/login")
+      },1500)
+    } catch (error) {
+      const err = getErrorMessage(error);
+      showToast("error", err);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  useEffect(()=>{
+    return ()=>{
+        if(timeoutRef.current) clearTimeout(timeoutRef.current)
+    }
+  },[])
+
+  return (
+    <Box>
+      <Typography variant="h5" component="p" sx={{ mb: 3 }}>
+        Change Password
+      </Typography>
+
+      <Box
+        component="form"
+        sx={{ display: "flex", flexDirection: "column", gap: "20px" }}
```

Comment:
We need to use the spacing system from MUI. gap:2.5 will result into gap:20px as 1 point resolves to 8px in the default theme. This is better as we will be able to update the spacing of theme easily and it will be reflected in our application.